### ISSUE TITLE
the FEED section of the author are inconvenient to watch, and confuses the user.

### ISSUE THREAD

**Issue by tehlit** [mix-blockchain/mix-acuity#18]
the picture and description of FEED should be different from other publications. The reader needs to understand the difference, where the description and the cover FEED, where you publish's this FEED

![2019-05-16_01-04-43 (2)](https://user-images.githubusercontent.com/28484248/57813760-fb289600-7779-11e9-89bb-410902bad246.png)
 
1)Feed: Title FEED
or Title Feed: Title FEED

2)I would remove the creation date of the FEED description.

3) add the number of posts in this FEED. This is important and useful.

4) the image should be concise and small. You need to scale them to standard sizes if it is large. The picture should take up little space on the screen, giving preference to the posts below.

5) description FEED. You need to limit for example 1000 characters maximum. Now you can make 30 000 characters and it will confuse the reader.

6) remove likes from feed description - it is not necessary

7) to remove the comment from the description is not necessary and confusing. Look 0xe41ca19df3110770afb11c5b07787471585fbb5dca704a661f1e136d1003177d

8) after describing the FEED, make a dividing line

**Comment by klmoney** [comment]
I completely agree with this suggestion. However note that currently the Description entered in the Publish Feed page does not persist unless there is an accompanying image. IOWs, the description is actually for the image and not for the feed it would appear. Much better if we can enter a description of the feed even without an image.
